client/controllers/cavokeclientcontroller.cpp

```diff
@@ -94,9 +101,26 @@ void CavokeClientController::startQmlApplication(
 }
 
 void CavokeClientController::exitApplication() {
-    // FIXME: not the best way, probably
     testWindowView.close();
     joinGameView.close();
     createGameView.close();
     startView.close();
 }
+void CavokeClientController::startQmlByPath(const QString& path) {
+    currentQmlGameModel =
+        new CavokeQmlGameModel(QUrl(path));
```

Comment:
Без этого не работает под виндой.

```suggestion
        new CavokeQmlGameModel(QUrl::fromUserInput(path));
```